Makefile

```
@@ -5,9 +5,9 @@ black = black src tests tests_common
 
 .PHONY: install
 install:
-	pip install -U setuptools pip
-	pip install -U -r requirements.txt -r requirements.dev.txt
-	pip install -e .
+	pip install -U setuptools pip uv
+	uv pip install -U -r requirements.txt -r requirements.dev.txt
+	uv pip install -e .
```

Might as well take the opportunity to fix this while we're at it.

```suggestion
	uv pip install -U --upgrade-strategy=eager -e . -c requirements.txt -r requirements.dev.txt
```